src/components/UserMenu/UserMenu.tsx

```diff
@@ -29,8 +29,12 @@ export function UserMenu({ userName, className }: UserMenuProps) {
     <div className={classNames(styles.wrapper, className)} onClick={onClick}>
       <div className={styles.head} title={userName}>
         <div className={styles.username}>{userName}</div>
-        <div className={styles.icon}>
-          <UserIcon width="1em" height="1em" />
+        <div className={styles.avatar}>
+          {image ? (
+            <Image {...image} className={styles.avatarImage} alt={userName} />
+          ) : (
+            <Image src="/images/icons/user.svg" className={styles.avatarImage} alt={userName} />
+          )}
```

Comment:
🧐
Если взять во внимание верхний коммент, то тут можно было бы что-то типа такого написать:
```suggestion
        <Image src={imageSrc || '/images/icons/user.svg'} className={styles.avatarImage} alt={userName} /> // и заглушку ещё круто было бы вынести в константу. Либо вычислять src в теле компонента как const src =  imageSrc || '/images/icons/user.svg'
```